<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SourcesForm</class>
 <widget class="QWidget" name="SourcesForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>664</width>
    <height>668</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="m_tabWidget">
     <property name="tabPosition">
      <enum>QTabWidget::South</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="m_tabHierarchy">
      <attribute name="title">
       <string>Files</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_2">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <attribute name="title">
       <string>Tab 2</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
